bb_behavior/plot/background.py

```diff
@@ -85,30 +85,30 @@ def generate_mode_images(gen, only_return_one=False, smoothing=0.95):
     if first_image is None:
         return None
 
-    is_float = not (first_image.dtype is np.integer)
+    is_float = not (first_image.dtype is int)
```

Comment:
won't work; np.integer is a base class and this will fail for np.uint now

Reply:
reverted back to np.integer